The server crashed after executing SQL with "where  userID-2>2"

<!-- Please describe your issue in English. -->

#### Can be reproduced?
Yes

#### Steps:
Run SQL below:
CREATE TABLE t1 (spID int,userID int,score smallint);
INSERT INTO t1 VALUES (1,1,1);
INSERT INTO t1 VALUES (2,2,2);
INSERT INTO t1 VALUES (2,1,4);
INSERT INTO t1 VALUES (3,3,3);
INSERT INTO t1 VALUES (1,1,5);
INSERT INTO t1 VALUES (4,6,10);
INSERT INTO t1 VALUES (5,11,99);

select * from t1 where  userID-2>2;


#### Expected behavior:
The SQL can run successfully and return correct results.

#### Actual behavior:
The server crashed immediately after running the SQL. Please see log below:

2021/09/17 17:26:13.970435 +0800 INFO frontend/mysql_cmd_executor.go:1360 query:select * from t1 where  userID-2>2
2021/09/17 17:26:13.973683 +0800 INFO frontend/mysql_cmd_executor.go:1125 time of Exec.Compile : 2.167541ms
+++++++++
		:0 π(userID,score,spID) -> σ(userID - 2 > 2) -> π(userID,score,spID) -> =>
	:2 merge -> =>
:1 merge -> sql output
+++++++++
2021/09/17 17:26:13.975453 +0800 INFO goetty@v1.11.0/application.go:257 session connected {"session-id": 2, "addr": "127.0.0.1:49177"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x440de75]

goroutine 1436 [running]:
github.com/RoaringBitmap/roaring/roaring64.(*Bitmap).Or(0xc00237aff0, 0xc0007d5400)
	/Users/anixjj/go/pkg/mod/github.com/!roaring!bitmap/roaring@v0.9.4/roaring64/roaring64.go:665 +0x35
matrixone/pkg/container/nulls.(*Nulls).Set(0xc000697b20, 0xc000697b18)
	/Users/anixjj/Workspace/matrixone/pkg/container/nulls/nulls.go:114 +0x6e
matrixone/pkg/sql/colexec/extend/overload.init.8.func3(0xc00237af50, 0xc00237a870, 0x5, 0x10, 0x0)
	/Users/anixjj/Workspace/matrixone/pkg/sql/colexec/extend/overload/minus.go:205 +0x437
matrixone/pkg/sql/colexec/extend/overload.BinaryEval(0xc00332ce10, 0x3, 0x3, 0x3e, 0x3, 0x0, 0x5abe440, 0xc002189780)
	/Users/anixjj/Workspace/matrixone/pkg/sql/colexec/extend/overload/binary.go:36 +0x19f
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0xc001f04600, 0xc0033e5a98, 0x400e974)
	/Users/anixjj/Workspace/matrixone/pkg/sql/colexec/extend/binary.go:53 +0xde
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0xc001f045d0, 0xc001f045d0, 0xc0033e5ab8)
	/Users/anixjj/Workspace/matrixone/pkg/sql/colexec/extend/binary.go:45 +0x38
matrixone/pkg/sql/colexec/restrict.Call(0xc002004c80, {0x549b880, 0xc001f04630})
	/Users/anixjj/Workspace/matrixone/pkg/sql/colexec/restrict/restrict.go:52 +0x52f
matrixone/pkg/vm.Run({0xc000204900, 0x4, 0x3}, 0xc00332db60)
	/Users/anixjj/Workspace/matrixone/pkg/vm/vm.go:49 +0x94
matrixone/pkg/vm/pipeline.(*Pipeline).Run(0xc0033e5f00, {0xc002188b40, 0x1, 0x1}, 0xc002004c80)
	/Users/anixjj/Workspace/matrixone/pkg/vm/pipeline/pipeline.go:93 +0x776
matrixone/pkg/sql/compile.(*Scope).Run(0xc002216400, {0x5a89140, 0xc002254d48})
	/Users/anixjj/Workspace/matrixone/pkg/sql/compile/scope.go:77 +0x699
matrixone/pkg/sql/compile.(*Scope).MergeRun.func1(0x0)
	/Users/anixjj/Workspace/matrixone/pkg/sql/compile/scope.go:92 +0x33
created by matrixone/pkg/sql/compile.(*Scope).MergeRun
	/Users/anixjj/Workspace/matrixone/pkg/sql/compile/scope.go:91 +0x412


#### Environment:
- Version or commit-id (e.g. v0.1.0 or 8b23a93): commit 80d19ee92a945f6caa3966dd8d74b54833c70831
- Hardware parameters:
- OS type:
- Others:

#### Configuration file:


#### Additional context:
- Error message from client:
- Server log:
- Other information:

--- a/pkg/container/nulls/nulls.go
+++ b/pkg/container/nulls/nulls.go
@@ -107,7 +107,7 @@ func (n *Nulls) Del(rows ...uint64) {
 }
 
 func (n *Nulls) Set(m *Nulls) {
-	if m != nil || m.Np != nil {
+	if m != nil && m.Np != nil {
 		if n.Np == nil {
 			n.Np = roaring.NewBitmap()
 		}

